src/renderer/components/ft-list-video/ft-list-video.js

```diff
@@ -327,6 +336,34 @@ export default defineComponent({
     this.checkIfWatched()
   },
   methods: {
+    getDeArrowTitle: async function() {
+      const data = await this.getDeArrowDataEntry()
+      if (data != null && data.titles.length > 0 && data.titles[0].locked) {
```

Comment:
Most titles won't be locked

```suggestion
      if (data != null && data.titles.length > 0 && (data.titles[0].locked || data.titles[0].votes > 0)) {
```

Reply:
Additionally as we are only using the first title, we probably don't need to store the rest of the titles array or even as an array at all. We can just cache the title string or `null` in the `getDeArrowDataEntry` function and then use the title here.